data_dir/scylla-dash-per-server-nemesis.3.1.json

```diff
@@ -73,7 +73,7 @@
             "3.1"
         ],
         "style": "dark",
-        "timezone": "browser",
+        "timezone": "utc",
```

Comment:
Looks good, we always to change it manually for matching with the log time.

Reply:
Changed it in any other configuration file as well